triage/2020-11-10.md

```diff
@@ -0,0 +1,39 @@
+# 2020-11-10 Triage Log
+
+A mixed week with improvements still outweighing regressions. Perhaps the biggest highlight was the move to compiling rustc crates [with the initial-exec TLS model](https://github.com/rust-lang/rust/pull/78201) which results in fewer calls to `_tls_get_addr` and thus faster compile times.
+
+Triage done by **@rylevick**.
+Revision range: [5cdf5b882da9e8b7c73b5cadeb7745cb68f6ff63..cf9cf7c923eb01146971429044f216a3ca905e06](https://perf.rust-lang.org/?start=5cdf5b882da9e8b7c73b5cadeb7745cb68f6ff63&end=cf9cf7c923eb01146971429044f216a3ca905e06&absolute=false&stat=instructions%3Au)
+
+1 Regressions, 2 Improvements, 2 Mixed
+
+#### Regressions
+
+[#78267](https://github.com/rust-lang/rust/issues/78267)
+- Small regression in [instruction counts](https://perf.rust-lang.org/compare.html?start=f92b931045dabb00b892519d3451cb41d41f2d31&end=8532e742fc6ec210fab69b8192190bc40c685912&stat=instructions:u) (up to 1.2% on `full` builds of `deeply-nested-async-check`)
+- This might be noise as this only affects one benchmark negatively, and that benchmark tends to be on the noisier side. 
+#### Improvements
+
+[#78280](https://github.com/rust-lang/rust/issues/78280)
+- Moderate improvement in [instruction counts](https://perf.rust-lang.org/compare.html?start=75f1db1102076e416e1154b241b4fc95c01c0d7b&end=89631663b7ad2d46d3e4f52bcfa7bee2be9eb82b&stat=instructions:u) (up to -1.3% on `incr-patched: new row` builds of `tuple-stress-opt`)
+
+[#78201](https://github.com/rust-lang/rust/issues/78201)
+- Large improvement in [instruction counts](https://perf.rust-lang.org/compare.html?start=fe8f02690804d5ee696bd3bca9515f5f71857e3b&end=25f6938da459a57b43bdf16ed6bdad3225b2a3ce&stat=instructions:u) (up to -7.2% on `incr-full` builds of `webrender-wrench-check`)
+- This change may produce similar performance gains in related tooling such as rustc and clippy.
```

Comment:
```suggestion
- This change may produce similar performance gains in related tooling such as rustdoc and clippy.
```

Reply:
And in fact we know for sure it helped rustdoc, there's `-doc` benchmarks on perf.rlo that improved too.